src/main/play-doc/operation/ClusterTroubleshooting.md

```diff
@@ -0,0 +1,54 @@
+# Cluster Troubleshooting
+
+Cluster partition is a failure that may occur when running a cluster based application. To recover from this situation ConductR utilizes Akka SBR (Split Brain Recovery). Akka SBR is a part of [Akka Typesafe Reactive Platform](http://www.typesafe.com/products/typesafe-reactive-platform) version `15v01p05`.
```

Comment:
It should be Typesafe Reactive Platform not Akka Typesafe Reactive Platform. We could say part of Akka in Typesafe Reactive Platform version or the likes, but the platform is TRP.